a10_octavia/controller/worker/tasks/cert_tasks.py

```diff
@@ -40,14 +40,24 @@ def execute(self, listener):
 
 class GetSSLCertData(task.Task):
     """Task to get SSL certificate data from Barbican"""
-
-    def execute(self, loadbalancer, listener, update_dict={}):
+     
+    @axapi_client_decorator
+    def execute(self, loadbalancer, listener, vthunder, update_dict={}):
         cert_data = None
         if listener and update_dict:
             listener.__dict__.update(update_dict)
         try:
             barbican_client = BarbicanACLAuth().get_barbican_client(loadbalancer.project_id)
             cert_data = utils.get_cert_data(barbican_client, listener)
+            if not cert_data.template_name:
+                client_ssl=self.axapi_client.slb.template.client_ssl.get(name=listener.id)
+                if client_ssl['client-ssl'].get('cert') or client_ssl['client-ssl'].get('key'):
```

Comment:
let's use and here to reduce issues for unexpected situations. (like only cert/key exist case)
```
if client_ssl['client-ssl'].get('cert') and client_ssl['client-ssl'].get('key'):
```

Reply:
DONE